src/landscape/components/LandscapeListMap.js

```diff
@@ -83,55 +73,169 @@ const LandscapesClusters = props => {
   }, [landscapes]);
 
   useEffect(() => {
-    if (!_.isEmpty(landscapesWithPosition)) {
-      const bounds = clusterRef.current?.getBounds?.();
-      if (bounds && bounds.isValid()) {
-        map.fitBounds(bounds);
-      }
+    if (!map) {
+      return;
     }
-  }, [map, landscapesWithPosition]);
 
-  return (
-    <MarkerClusterGroup
-      ref={clusterRef}
-      maxClusterRadius={40}
-      showCoverageOnHover={false}
-      iconCreateFunction={cluster => {
-        return L.divIcon({
-          className: 'landscape-list-map-cluster-icon',
-          iconSize: new L.Point(40, 40),
-          html: `<div>${cluster.getChildCount()}</div>`,
+    map.addSource('landscapes', {
+      type: 'geojson',
+      data: {
+        type: 'FeatureCollection',
+        features: landscapesWithPosition.map(landscape => ({
+          type: 'Feature',
+          geometry: {
+            type: 'Point',
+            coordinates: landscape.position.reverse(),
+          },
+          properties: {
+            ...landscape.data,
+          },
+        })),
+      },
+      cluster: true,
+      clusterMaxZoom: 14,
+      clusterRadius: 50,
+    });
+
+    map.addLayer({
+      id: 'clusters',
+      type: 'circle',
+      source: 'landscapes',
+      filter: ['has', 'point_count'],
+      paint: {
+        'circle-color': '#ff580d',
+        'circle-radius': 15,
+        'circle-stroke-width': 5,
+        'circle-stroke-color': '#ff580d',
+        'circle-stroke-opacity': 0.5,
+      },
+    });
+
+    map.addLayer({
+      id: 'cluster-count',
+      type: 'symbol',
+      source: 'landscapes',
+      filter: ['has', 'point_count'],
+      layout: {
+        'text-field': ['get', 'point_count_abbreviated'],
+        'text-font': ['DIN Offc Pro Medium', 'Arial Unicode MS Bold'],
```

Comment:
question: where does this font stack come from?

Reply:
I have to check I think ideally we should use the same fonts as the rest of the site, so I will see if I can remove them